cmd/cluster/context.go

```diff
@@ -418,7 +417,7 @@ func (o *contextOptions) generateContextData() (*contextData, []error) {
 			}
 			queryTxt := query.Build()
 			data.DyntraceEnvURL = hcpCluster.DynatraceURL
-			data.DyntraceLogsURL = dynatrace.GetLinkToWebConsole(hcpCluster.DynatraceURL, 10, base64.StdEncoding.EncodeToString([]byte(queryTxt)))
+			data.DyntraceLogsURL = dynatrace.GetLinkToWebConsole(hcpCluster.DynatraceURL, 10, queryTxt)
```

Comment:
Can we make the "since" variable either 
1. An optional parameter for the function entirely because we donot need that here to be passed as 10 hrs. We can go with default value OR
2. Move the `10` to a separate variable and then pass it here to give clarity what the paramter us

Reply:
The 10 is defaulf value. I've not edited it.
The since variable is present in _logsCmd's_ GetLinkToWebConsole func